modules/doobie/src/main/scala/com/evolutiongaming/smetrics/TransactorMetered.scala

```diff
@@ -0,0 +1,29 @@
+package com.evolutiongaming.smetrics
+
+import java.sql.Connection
+
+import cats.data.Kleisli
+import cats.implicits._
+import com.evolutiongaming.catshelper.BracketThrowable
+import doobie.free.connection.ConnectionOp
+import doobie.util.transactor.{Interpreter, Transactor}
+
+object TransactorMetered {
+
+  def apply[F[_]: BracketThrowable: MeasureDuration](
+    transactor: Transactor[F],
+    dbMetrics: DbMetrics[F]
```

Comment:
I think it makes sense to call it `metrics` in this scope

Reply:
fixed